Libs/MRML/Core/vtkMRMLVolumeNode.cxx

```diff
@@ -669,6 +669,43 @@ const char* vtkMRMLVolumeNode::ComputeScanOrderFromIJKToRAS(vtkMatrix4x4 *ijkToR
   }
 }
 
+//----------------------------------------------------------------------------
+// Checks that a vtkImageData has origin (0,0,0), spacing (1,1,1), and an identity direction matrix
+bool imageDataHasIdentityAffine(vtkImageData *imageData)
+{
+  // Should we be using vtkMathUtilities::FuzzyCompare here?
+  // I am thinking no because we vtkImageData affine should be *exactly* identity
+  // (we don't want "slightly" undefined behavior from slicer).
```

Comment:
Note my question here. I believe we should be checking for exact equality of floats throughout in this function. Just want to double check that everyone agrees.

Reply:
I agree, exact check is appropriate here (especially because both 0.0 and 1.0 can be represented exactly as float).